content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
+        このARIAオーサリングプラクティスガイド（APG）への導入では、APGの目的、前提となる知識、およびその構成について説明します。
       </p>
 
       <section id="purpose">
-        <h2>Purpose of the APG</h2>
+        <h2>APGの目的</h2>
         <p>
-          The APG synthesizes requirements from multiple accessibility and web technology specifications into guidance for developing web experiences for people who rely on assistive technologies and keyboard interfaces.
-          It includes a library of accessible user interface patterns, example implementations of the patterns, and guidance sections that explain how to:
+          APGは、複数のアクセシビリティおよびウェブ技術仕様からの要件を統合し、支援技術やキーボードインターフェースに依存する人々のためのウェブ体験を開発するためのガイダンスを提供します。
+          これには、アクセシブルなユーザーインターフェースのパターンのライブラリ、パターンの実装例、および以下の方法を説明するガイダンスセクションが含まれます：
         </p>
         <ul>
           <li>
-            Make common user interface patterns accessible by using features of the
-            <a class="specref" href="#abstract">Accessible Rich Internet Application (ARIA) specification</a>
-            in combination with HTML, CSS, JavaScript, and other web technologies.
+            <a class="specref" href="#abstract">Accessible Rich Internet Application (ARIA) 仕様</a>の特徴をHTML、CSS、JavaScript、およびその他のウェブ技術と組み合わせて使用することにより、一般的なユーザーインターフェースのパターンをアクセシブルにする。
           </li>
           <li>
-            Use ARIA, HTML, and CSS to implement fundamental accessibility practices, such as providing accessible names, enabling keyboard interaction, conveying page structure to assistive technologies, and supporting high contrast operating system settings.
+            ARIA、HTML、およびCSSを使用して、アクセシブルな名前を提供し、キーボードインタラクションを有効にし、支援技術にページの構造を伝え、高コントラストのオペレーティングシステム設定をサポートするなど、基本的なアクセシビリティ実践を実装する。
           </li>
         </ul>
         <p>
-          The primary aim of the APG is to increase the availability of robust accessible experiences by building broad understanding  and encouraging adoption of web accessibility practices that:
+          APGの主な目的は、関連するウェブアクセシビリティの実践の理解を広げ、その採用を促進することによって、堅牢なアクセシブルな体験の提供を増やすことです：
         </p>
         <ul>
-          <li>Are consistent with the intent of ARIA, HTML, CSS, and related specifications.</li>
-          <li>Support assistive technology interoperability, which is assessed by the <a href="https://aria-at.w3.org/">ARIA and Assistive Technologies Project</a>.</li>
-          <li>Promote practical techniques for developing modern GUIs that both support usability for people with disabilities and conform with <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> requirements.</li>
+          <li>ARIA、HTML、CSS、および関連する仕様の意図と一致している。</li>
+          <li>支援技術の相互運用性をサポートし、<a href="https://aria-at.w3.org/">ARIA and Assistive Technologies Project</a>によって評価される。</li>
+          <li>障害を持つ人々の使用可能性をサポートし、<a href="https://www.w3.org/TR/WCAG21/">WCAG</a>の要件に準拠する、現代のGUIを開発するための実用的な技術を推進する。</li>
```

Comment:
usabilityは使用性くらいに訳しておいた方がいいかなと思いました。
JISでの訳語が「使用性」ですし、「使用可能性」としてしまうと「そもそも使えるか」のニュアンス、つまりアクセシビリティのニュアンスと近くなり焦点がぼやけてしまうのかなと…。

techniquesは技術と訳すとやや堅く、テクニックと訳出した方が自然でしょうか。

```suggestion
          <li>障害を持つ人々の使用性をサポートし、<a href="https://www.w3.org/TR/WCAG21/">WCAG</a>の要件に準拠する、現代のGUIを開発するための実用的なテクニックを推進する。</li>
```

あまり自信ないです！